### PR TITLE
Finalize Open Day Page

### PR DESCRIPTION
**Pull request by TheUltimateKeval**
Changes made:
- Added remaining events to the page
- Addressed navbar overflow issue
- Made a proper poster image for RPS exhibit

I am thinking to keep Open Day page on the navbar for some time (a month or so) since it was a big event.
Later we can move it to a special section on the events page.

### CODE REVIEW

File: css/common.css [hunk 1]
```diff
@@ -90,7 +90,7 @@ nav {
 }
 
 
-@media only screen and (max-width: 900px) {
+@media only screen and (max-width: 1000px) {
```
Comment (mrigankpawagi) [inline css/common.css:93]: There is still some margin of 25-50 pixels. For how long are we planning to keep *Open Day* on the menu?

**Review by mrigankpawagi — APPROVED** [review]

**Review by mrigankpawagi — DISMISSED** [review]

**Comment by mrigankpawagi** [comment]
LGTM otherwise

File: css/common.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (TheUltimateKeval) [reply css/common.css:93]: It seems to work fine on my system. Probably some browser related differences. I'll add some extra margin for safety.

[the remark this answers is not in the document]
Comment (TheUltimateKeval) [reply css/common.css:93]: I was planning to keep Open Day on the menu for the month of march.

[the remark this answers is not in the document]
Comment (mrigankpawagi) [reply css/common.css:93]: Perfect!

**Pull request merged**